src/aihawk_easy_applier.py

```diff
@@ -21,6 +21,10 @@
 
 import src.utils as utils
 from loguru import logger
+import threading
+
+# Create a thread-local storage
+local = threading.local()
```

Comment:
why is thread local required? here

Reply:
yes, why?

Reply:
I was solving for some concurrency errors that seem to have disappeared.

updated with new push.